src/functions.md

```diff
@@ -6,101 +6,79 @@ Functions are where actual work happens. There are built-in functions and user-d
 
 Declaration of functions works like so:
```

Comment:
can we be more explicit about this declaration working inside or outside handlers?

Reply:
Can you give me an example of something you think I wasn't clear on? I thought this section was perfectly clear myself. :/

Reply:
Still waiting on this one @depombo

Reply:
So if I remember correctly, the following does not work outside handlers right now:
```
fn functionName (firstArgument: argumentType, secondArgument: anotherType): returnType {
  ...statements...
}
```

Reply:
So we have [a whole section of the test suite](https://github.com/alantech/alan/blob/master/bdd/spec/008_function_spec.sh#L12) to confirm that it works?